group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroup.java

```diff
@@ -653,12 +653,15 @@ public DeadlineAndEpoch metadataRefreshDeadline() {
      * @param memberId          The member id.
      * @param groupInstanceId   The group instance id.
      * @param memberEpoch       The member epoch.
+     * @param isTransactional   Whether the offset commit is transactional or not. It has no
+     *                          impact when a consumer group is used.
      */
     @Override
     public void validateOffsetCommit(
         String memberId,
         String groupInstanceId,
-        int memberEpoch
+        int memberEpoch,
+        boolean isTransactional
```

Comment:
Will we add the checks for transactional commits in the next PR? Or do you think we don't need them for new ConsumerGroup?

Reply:
I thought that we don't need it for the "new consumer" groups. My reasoning is that they should not use old versions of the TxnOffsetCommit API if they use the new protocol.